kolibri/plugins/coach/assets/src/views/common/resourceSelection/subPages/SelectionIndex.vue

```diff
@@ -23,6 +23,7 @@
         {{ selectFromBookmarks$() }}
       </div>
       <KButton
+        v-if="bookmarksCount > 0"
```

Comment:
This will hide the search button unless the user has bookmarks. I am unsure how this is fixing the issue?

I think you probably intended to do the same if statement here as below, conditionalised on channels.length.